spec/runspec.vim_spec.rb

```diff
@@ -42,15 +47,54 @@ def runspec(command)
       end
 
       it "finds a test with the same name" do
-        FileUtils.touch("test/foo_test.rb")
+        touch("test/foo_test.rb")
+        touch("foo.rb")
         expect(runspec('SpecPath("foo.rb")')).to eq("test/foo_test.rb")
       end
 
       it "finds a test with the most similar name" do
-        FileUtils.mkdir("test/unit")
-        FileUtils.touch("test/unit/user_test.rb")
+        touch("app/models/user.rb")
+        touch("test/unit/user_test.rb")
         expect(runspec('SpecPath("app/models/user.rb")')).to eq("test/unit/user_test.rb")
       end
+
+      it "finds a test from a lib directory" do
+        touch("lib/app_name/something.rb")
+        touch("test/app_name/something_test.rb")
```

Comment:
The reason why I create both the test and corresponding implementation here is because an earlier version of `s:Hunt` would have issues because I relaxed the first file match so that it could find `autoload/runspec.vim` from `spec/runspec.vim_spec.rb` but it was too eager to find and match the path that was passed in. For this reason (although for reasons unknown to me, the specs never caught this error), I removed that bit from `s:Hunt` which means you can't use the toggle command to hop between `autoload/runspec.vim` and `spec/runspec.vim_spec.rb` - it'll only work one way.